src/System.Net.Http.WinHttpHandler/ref/Configurations.props

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netcoreapp;
+      uap;
```

Comment:
I believe it is, by looking at some pre-dev/eng packages it seems to have a netstandard1.3 reference assembly.